modules/account-lib/src/coin/cspr/transactionBuilder.ts

```diff
@@ -70,26 +109,51 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return this;
   }
 
+  /**
+   * Set the transaction validDuration
+   *
+   * @param {string} validDuration The transaction validDuration
+   * @returns {TransactionBuilder} This transaction builder
+   */
+  duration(validDuration: string): this {
```

Comment:
nit: replace `duration` with `expiration`
```suggestion
  expiration(expirationTime: string): this {
```